explorer/app/components/Project/components/Section/components/SectionTitle/sectionTitle.tsx

```diff
@@ -3,15 +3,16 @@ import { Typography } from "@mui/material";
 import React from "react";
 
 interface Props {
+  className?: string;
   title: string;
 }
 
-export const SectionTitle = ({ title }: Props): JSX.Element => {
+export const SectionTitle = ({ className, title }: Props): JSX.Element => {
   return (
     <Typography
+      className={className}
       color="ink"
       component="h3"
-      fontWeight={600}
```

Comment:
Font weight will be resolved with [ticket #287](https://github.com/clevercanary/data-browser/issues/287)